std_msgs/Header header
std_msgs/Int32 n
ros_object_detector/DetectedObject[] objects
sensor_msgs/Image image
sensor_msgs/PointCloud2 pc
geometry_msgs/TransformStamped camera_world_tf
geometry_msgs/TransformStamped world_camera_tf
